_translations/en.yml

```diff
@@ -367,11 +385,47 @@ en:
+    tenminutes: "Why do I have to wait for confirmation?"
+    tenminutestxt1: >
+      Receiving notification of a payment is almost instant with Bitcoin. However, there
+      is a delay before the network begins to
+      confirm your transaction by including it in a block.
+      A confirmation means that
+      there is a consensus on the network that the bitcoins you received
+      haven't been sent to anyone else and are considered your property.
+      Once your transaction has been included in one block, it will
+      continue to be buried under every block after it, which will
+      exponentially consolidate this consensus and decrease the risk of
+      a reversed transaction.
+      Each confirmation takes between a few seconds and 90 minutes, with
+      10 minutes being the average. If the transaction pays too low a
+      fee or is otherwise atypical, getting the first confirmation can
+      take much longer.  Every user is free to determine at what
+      point they consider a transaction sufficiently confirmed, but <a href="#you-need-to-know##instant">6 confirmations</a>
+      is often considered to be as safe as waiting 6 months on a credit
+      card transaction.
+
     fee: "How much will the transaction fee be?"
-    feetxt1: "Most transactions can be processed without fees, but users are encouraged to pay a small voluntary fee for faster confirmation of their transactions and to remunerate miners. When fees are required, they generally don't exceed a few pennies in value. Your Bitcoin client will usually try to estimate an appropriate fee when required."
-    feetxt2: "Transaction fees are used as a protection against users sending transactions to overload the network. The precise manner in which fees work is still being developed and will change over time. Because the fee is not related to the amount of bitcoins being sent, it may seem extremely low (0.0005 BTC for a 1,000 BTC transfer) or unfairly high (0.004 BTC for a 0.02 BTC payment). The fee is defined by attributes such as data in transaction and transaction recurrence. For example, if you are receiving a large number of tiny amounts, then fees for sending will be higher. Such payments are comparable to paying a restaurant bill using only pennies. Spending small fractions of your bitcoins rapidly may also require a fee. If your activity follows the pattern of conventional transactions, the fees should remain very low."
+    feetxt1: >
+      Transactions can be processed without fees, but users are
+      encouraged to pay a voluntary fee for faster confirmation of
+      their transactions and to remunerate miners. When fees are
+      required, they currently don't exceed a few pennies in value,
+      but this may increase over time. Your
+      Bitcoin client will pay what it thinks is an appropriate fee.
```

Comment:
@harding How about:

~~~
Your Bitcoin client will display what it think is an appropriate fee.
~~~

I feel like the current text suggests wallet will pay whatever fee they please in the background without informing the user. In reality, wallets typically show the transaction fee somewhere in the UI before the user clicks "Send".